### ISSUE TITLE
[VOC VIZ] Create New VOC Map View

### ISSUE THREAD

**Issue by rkassa** [globaldothealth/list#2041]
**Description:**
- Create new nav item in Maps for VOCs and its relevant map view
- New Map View will display color coded data from #2040 
- Map view should default to `any_variant_info` column data
- Filters should be available to toggle view to display other variant columns or personal/demographic data

![b 1 1 7](https://user-images.githubusercontent.com/201428/127510906-d1baf167-2443-42c8-88f6-aae8cb44024a.jpg)

**Comment by rkassa** [comment]
This would be a new map view from the main nav:

![image](https://user-images.githubusercontent.com/201428/128363510-0f8f13e0-7563-4b01-920e-dd7995edc050.png)

**Comment by Mougk** [comment]
I would suggest to the right of coverage. Maybe use variant reporting rather than variant public reporting (shorter).

**Comment by joe-brilliant** [comment]
Recapping notes from sync / slack cc @rkassa @outbreakprepared 

Add the following fields to hover state: 

- [ ] source URL
- [ ] date checked
- [ ] reported breakthrough cases

- [ ] Group Variants of Concern and Variants of Interest in menu sections, with VoCs first, and including the Greek names for VoCs (e.g. Delta) as defined by the WHO https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/

- [ ] add some basic logic to merge all the Delta "sub"variants

- [ ] Breaking out USA into states

- [ ] Add in some data-processing logic so that variants of concern/interest can iteratively be updated as WHO makes updates. For example, 'Mu' got added as a variant of interest

**Comment by joe-brilliant** [comment]
@rkassa @outbreakprepared @ryanfitzgerald19 let's review the latest version and go through these items above and determine a go-live date to back out from as at top agenda item for tomorrow's sync

**Comment by maciekz1996** [comment]
Hi @ryanfitzgerald19, I'm having trouble understanding this bullet point: `Group Variants of Concern and Variants of Interest in menu sections, with VoCs first, and including the Greek names for VoCs (e.g. Delta) as defined by the WHO`. @joe-brilliant said that you could help me understand that. 

Currently all VoCs from the Google spreadsheet are in one big dropdown (see screenshot below). The order in which those VoCs appear is the same as the order in the spreadsheet. Please let me know how do you think it should be split into sections? Maybe creating separate dropdown for VOIs? It is possible to sort all VoCs and VoIs coming from spreadsheet into different categories, I'm just not sure how to display them on the map. 

<img width="759" alt="Screen Shot 2021-09-28 at 13 55 15" src="https://user-images.githubusercontent.com/44740625/135083932-260285a4-f126-49eb-89e2-e917628246b0.png">

Also I'm not sure about this one: `add some basic logic to merge all the Delta "sub"variants`. Can you also explain it a little bit more?

Thank you in advance!

**Comment by joe-brilliant** [comment]
Hi @ryanfitzgerald19 @outbreakprepared @Mougk bumping this last comment / question from @maciekz1996. Could you help provide some feedback and guidance to move this forward next week while I'm out? I like the idea of separate dropdowns for Variants of Concern and Variants of Interest as a simple and intuitive first approach. I think we will also need input to decide on what should be included in the popups, latest prototype here: http://dev-vocviz-sample.s3-website.us-east-2.amazonaws.com/

**Comment by maciekz1996** [comment]
@joe-brilliant That's my bad, the new help text was added on Friday but I didn't deploy it correctly to AWS do that's the reason you couldn't see the updated content. I'm working on this right now. I also reached out to @rkassa as I'm experiencing one issue during deployment process. When it comes to the letters being misplaced, I'm not sure if I understand this correctly. I think it would be best to go through it on today's call
